iron-list.html

```diff
@@ -232,7 +232,6 @@
     <style>
       :host {
         display: block;
-        position: relative;
```

Comment:
Potentially a breaking change if folks rely on this property to stack things on top.